proto/xudrpc.proto

```diff
@@ -410,6 +425,19 @@ message Order {
   double hold = 10 [json_name = "hold"];
 }
 
+message Trade {
+  // The order id of the maker order involved in this trade.
```

Comment:
This comment can just say "The maker order involved in this trade" - same for the taker order below.